packages/nextjs/app/builders/page.tsx

```diff
@@ -61,41 +52,101 @@ const Builders: NextPage = () => {
+          src="/blur_yellow.svg"
+          alt="Blurred Yellow Background"
+          className="absolute -top-[10%] -left-[40%] w-200 h-200 blur-2xl"
+          width={2000}
+          height={2000}
+        />
       </div>
-      <div className="max-w-screen-lg mx-auto mt-10 p-6 bg-base-300 shadow-md rounded-lg">
-        <div className="container mx-auto px-4 py-8">
-          <div className="mb-12">
-            <h2 className="text-2xl font-semibold text-center mb-6">Mentors</h2>
-            <div className="grid grid-cols-3 gap-6">
-              {mentors.map((mentor, i) => (
-                <MentorPicture person={mentor} key={i} />
-              ))}
+
+      <div className="absolute w-full h-full pointer-events-none">
+        <Image
+          src="/blur_blue.svg"
+          alt="Blurred Blue Background"
+          className="absolute -bottom-[10%] -right-[40%] blur-2xl"
+          width={2000}
+          height={2000}
+        />
+      </div>
+
+      <div className="flex flex-col gap-12 max-w-7xl justify-between w-full mt-10 sm:mt-18 z-10">
+        {/* Mentor Section */}
+        <div className="flex flex-col px-10 gap-4 w-full ">
+          {/* Mentor Header */}
+          <div className="flex gap-2 sm:gap-5 items-center justify-between">
+            <div className="tracking-wider flex flex-col items-start">
+              <h1 className="font-semibold text-xl sm:text-2xl md:text-3xl">Our Mentors..</h1>
+              <p className="text-sm md:text-base -mt-1 text-zinc-500">Meet our batch mentors..</p>
+            </div>
+
+            <div className="flex justify-center items-center">
+              <p className="flex border dark:border-zinc-700 border-zinc-400 py-3 px-10 rounded-xl font-semibold gap-2 justify-center text-cyan-500">
+                <span className="font-medium flex-shrink-0">Counter: {mentors.length}</span>
+              </p>
             </div>
           </div>
 
-          <div>
-            <h2 className="text-2xl font-semibold text-center mb-6">Builders 🏗</h2>
-            <p className="text-sm font-bold text-center mb-8">Builders that checkedIn: {buildersCheckedIn}</p>
-            <div className="grid grid-cols-5 gap-20">
+          {/* Mentor Cards */}
+          <div className="grid grid-flow-row sm:grid-flow-col gap-10 lg:flex-row justify-evenly w-full border dark:border-zinc-700 border-zinc-400 rounded-xl pt-10 pb-6 px-8 sm:overflow-x-scroll scrollbar-primary">
+            {mentors.map((mentor, i) => (
+              <BuilderCard mentor={mentor} key={i} />
+            ))}
+          </div>
+        </div>
+
+        {/* Builder Section */}
+        <div className="flex flex-col px-10 gap-4 w-full mb-10 z-10">
+          {/* Builder Header */}
+          <div className="flex gap-2 sm:gap-5 items-center justify-between">
+            <div className="tracking-wider flex flex-col items-start">
+              <h1 className="font-semibold text-xl sm:text-2xl md:text-3xl">Our Builders..</h1>
```

Comment:
I'm experiencing the same issue here, but no matter how large the window (on chrome, firefox) is, I can't see or scroll to the left card. Additionally, when trying to scroll horizontally, I often end up navigating back on my Mac. Switching to vertical scrolling might be a better solution.

![Screenshot 2024-09-26 at 19 26 55](https://github.com/user-attachments/assets/8640a9e6-7749-421e-b83f-b71a3ba01fd4)

Reply:
Okay so i wasn't able to reproduce any of these issues on my chrome but as soon as i switched to firefox, I realized everything looks pretty different... let me figure out what's exactly wrong

Reply:
modified the layout to fix the issue... should be fine now with vertical layout

Reply:
although i discovered a new bug where the blur css property does not work properly on firefox, so my bg-ellipse blur effect was looking bad on it. i couldn't find any proper fix around it so i just tweaked the opacity to make it look consistent. for some reason, everything looked and worked perfect on chrome since i was testing on it rigorously but didn't check cross-browser support. will keep it in mind next time!